extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/CommonCacheNotifier.java

```diff
@@ -140,6 +140,11 @@ public void start()
 
               LOG.debug(callerName + ":Received responses for cache update notifications.");
             }
+            catch (InterruptedException e) {
+              LOG.noStackTrace()
```

Comment:
should this be emitted as an alert?

Reply:
I guess we could just log an INFO here instead.

Reply:
makes sense to me.